rclrs/src/node/builder.rs

```diff
@@ -142,28 +152,97 @@ impl NodeBuilder {
         self
     }
 
+    /// Sets flag for using global arguments.
+    /// If false then only use arguments passed to this builder,
+    /// otherwise use global arguments also.
+    ///
+    /// # Example
+    /// ```
+    /// # use rclrs::{Context, Node, NodeBuilder, RclrsError};
+    /// let args = ["--ros-args", "--remap", "my_node:__node:=your_node"]
+    ///   .iter()
+    ///   .map(|s| s.to_string())
+    ///   .collect::<Vec<_>>();
+    /// let context = Context::new(args)?;
+    /// // Use global arguments
+    /// let node = context
+    ///   .create_node_builder("my_node")
+    ///   .use_global_arguments(true)
+    ///   .build()?;
+    /// assert_eq!(node.name(), "your_node");
+    ///
+    /// // Not use global arguments
+    /// let node = context
+    ///   .create_node_builder("my_node")
+    ///   .use_global_arguments(false)
+    ///   .build()?;
+    /// assert_eq!(node.name(), "my_node");
+    /// # Ok::<(), RclrsError>(())
+    pub fn use_global_arguments(mut self, enable: bool) -> Self {
+        self.use_global_arguments = enable;
+        self
+    }
+
+    /// Sets node specific command line arguments.
+    ///
+    /// Command line arguments can mix ROS specific and not specific arguments.
+    /// ROS specific arguments are wrapped between `--ros-args` and `--`.
+    /// Note that `--ros-args` followed by only ROS specific arguments is valid pattern.
+    ///
+    /// below arguments examples are valid:
+    ///   --ros-args -r from:=to -- user_arg0 user_arg1
+    ///   --ros-args -p name:=value
+    ///    
+    /// # Example
+    /// ```
+    /// # use rclrs::{Context, Node, NodeBuilder, RclrsError};
+    /// let context = Context::new([])?;
+    /// let node_args = ["--ros-args", "--remap", "my_node:__node:=your_node"]
+    ///   .iter()
+    ///   .map(|s| s.to_string())
+    ///   .collect::<Vec<_>>();
+    /// let node = context
+    ///   .create_node_builder("my_node")
+    ///   .arguments(node_args)
+    ///   .build()?;
+    /// assert_eq!(node.name(), "your_node");
+    /// # Ok::<(), RclrsError>(())
+    /// ```
+    pub fn arguments(mut self, arguments: impl IntoIterator<Item = String>) -> Self {
+        self.arguments = arguments.into_iter().collect();
+        self
+    }
+
+    /// Enables rosout.
+    /// If false, rosout logging is disabled.
+    pub fn enable_rosout(mut self, enable: bool) -> Self {
```

Comment:
Do you know if this actually affects logs emitted by the `rcl` layer? Or does it currently not have any effect at all?

Reply:
I believe it currently has no effect.

Reply:
As for now, there is no effect in ros2_rust.
However, as far as I know, this flag affects whether logging features publish to `rosout` topic or not.
For the future, this should retain and put comments to tell this is nothing to do for now, I think.